lib/java/deploy/MavenPerBranchSpringBootDeploymentGoal.ts

```diff
@@ -99,8 +126,10 @@ export function executeMavenPerBranchSpringBootDeploy(projectLoader: ProjectLoad
         try {
             const deployment = await projectLoader.doWithProject({ credentials, id, readOnly: true },
                 project => deployer.deployProject(goalInvocation, project));
-            await goalInvocation.addressChannels(`Deployed \`${id.owner}/${id.repo}/${goalInvocation.sdmGoal.branch} [${
+            const deploymentKey = `${id.owner}/${id.repo}/${goalInvocation.sdmGoal.branch}`;
+            await goalInvocation.addressChannels(`Deployed \`${deploymentKey} [${
```

Comment:
Seems odd to break inside a `${}`, although prettier certainly likes to do that.

Reply:
Reformatted